tests/network/vmi_multus.go

```diff
@@ -672,7 +671,7 @@ var _ = SIGDescribe("[Serial]Multus", func() {
                     ip addr add %s dev ep1
                     ip addr add %s dev ep2
                 `, ep1Cidr, ep2Cidr, ep1CidrV6, ep2CidrV6)
-				agentVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskFedoraTestTooling), userdata)
+				agentVMI := libvmi.NewFedora(libvmi.WithCloudInitNoCloudUserData(userdata, false))
```

Comment:
IIUC, `agentVMI` was a `NewTestToolingFedora` but would now be a `NewFedora` . Is this change intentional ? If so, would you elaborate on its motivation ?

Reply:
No, it was not intentional. I'll fix it. However please note that for several months now, the two entities are [one and the same](https://github.com/kubevirt/kubevirt/blob/main/tests/libvmi/factory.go#L37).

Reply:
wow. OK. Isn't that something we should somehow fix ? I mean... it's misleading at best. 

Let's say you have functions `NewSalad` and `NewSaladWithLotsOfNuts` . These - for whatever reason - are the same.
If I were allergic to nuts, I would happily consume the `NewSalad`, thinking myself very safe, and then go and have an anaphylaxis attack.

Reply:
Yes, I found it quite surprising myself. I don't see if this question was raised when https://github.com/kubevirt/kubevirt/commit/36aba3ec0d7feecc379bd157ca3e424c14bf2e48#diff-6744b3e54011de85527b1a5b5deb98fbc05a60aa7576c644315d5dc25ebcb74bL40 was discussed. It is also not clear to me what is the right solution: is it to re-split between vanilla Fedora and Fedora-with-nuts? Or drop all traces of `NewTestToolingFedora` from the code?

I tend for the lazier second option. Anyway, would you not agree that it is unrelated to this tiny PR?

Reply:
100 % not related :) 

I actually think the *right* thing (take it with a pinch of salt ...) is to have the 2 of them. But it falls to the user (person writing the tests) to know which to use. I don't think it's too hard to decide: "need tools?" => use the test tooling one. 

But hey, let's discuss this somewhere down the line on a PR dedicated to this.